src/js/themes/base.js

```diff
@@ -418,6 +418,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           },
         },
       },
+      // gap: undefined,
```

Comment:
Should the default `small` from Button.js get moved here to the theme?

Reply:
I was debating on this as well and removing from the prop I just wasn't sure